src/fragments/lib/in-app-messaging/integrate-your-application/js/install-dependencies.mdx

```diff
@@ -19,5 +13,5 @@ import js0 from '/src/fragments/lib/in-app-messaging/integrate-your-application/
 </Callout>
 
 ```bash
-npm install @aws-amplify/ui-react
+npm install aws-amplify @aws-amplify/ui-react-notifications
```

Comment:
Just to confirm, do we reference _@aws-amplify/ui-react_ anywhere in the example code still? Seems likely there is still the style import, if so, believe we should include both packages

Reply:
+1

Reply:
Yes, we use styles from `@aws-amplify/ui-react`. Will update this install script.